features/step_definitions/journey_test_steps.rb

```diff
@@ -877,10 +877,10 @@ def filter_to_check(filter_name,filter_value,filter_exist)
 end
 
 Then /I am on a page with '(.*)' in search summary text$/ do |value|
+  find(:xpath, "//*[@class='search-summary']/em[1]").text().should == normalize_whitespace(value)
```

Comment:
There's an explanation in the commit message. It's not the first time we run into this, but I don't think a comment with "find should be the first check" in this step would help with similar issues in others (i.e. it won't be any more noticeable than a commit message)